lib/spack/docs/binary_caches.rst

```diff
@@ -0,0 +1,99 @@
+.. _binary_caches:
+
+Build caches
+============================
+
+Some sites may encourage users to set up their own test environments
+before carrying out central installations, or some users prefer to set
+up these environments on their own motivation. To reduce the load of
+recompiling otherwise identical package specs in different installations,
+created build artifacts can be put into build cache tarballs, uploaded to 
+your spack mirror and then downloaded and installed by others.
+
+
+Creating build cache files
+--------------------------
+
+A compressed tarball of an installed package is created.
+Tarballs are created for all of its link and run 
+dependency packages as well.
+Compressed tarballs are signed with gpg and 
+signature and tarball and put in a ".spack" tarfile. 
```

Comment:
There's no need to mention that it's a tarfile. Also, the wordwrapping here is odd. Consider reflowing?